tests/test_api.py

```diff
@@ -84,6 +59,10 @@ def test_train(self, bliss_client, monkeypatch):
         monkeypatch.setattr(api, "_train", mock_train)
         bliss_client.train("test", training={"n_epochs": 1})
 
+    def test_train_on_cached_data(self, cfg, bliss_client, monkeypatch):
+        monkeypatch.setattr(api, "_train", mock_train)
+        bliss_client.train_on_cached_data("test.pt", 4, 4, None, None, "test_pretrained.pt")
```

Comment:
Why `"test.pt"` / these arguments? In general, the goal of test_api is to mimic how an end user would call our `BlissClient` functions, so I would want to use arguments like

`("output/test.pt", 4, 4, [1])`

(Did you include one more argument here than `train_on_cached_data` expects?)

Reply:
Whoops, I thought it took both val and test indices but I guess not. None of these arguments are actually going to be used since `_train` is mocked, so I just filled in the arguments arbitrarily

Reply:
Ah, the extra argument was added in Sawan's multi-band PR yesterday, so it does take test indices now. I also updated the arguments and how `test_train` and `test_train_on_cached_data_work` altogether - see comment below.